### PR TITLE
CSS basics

### PR DESCRIPTION
**Pull request by MahtaMirzaei**
Related: #2

### CODE REVIEW

**Review by mkermani144 — CHANGES_REQUESTED** [review]
Please reduce the html file to a very simple one, containing a class or id for the html tag (as mentioned in the issue). Also remove redundant lines of code from the css file.

In addition, I see your html file from the previous issue in the changes of this PR, which is unexpected. Please investigate the cause of the file inclusion.

**Comment by MahtaMirzaei** [comment]
I did the corrections you asked for, but I'm not certain about the cause of my HTML file being mentioned in this PR. I pushed my changes on the HTML branch, then I made the CSS branch, checked out to it and the HTML files weren't in my folder until you allowed the HTML branch to merge to main.

Did I do it right?

**Comment by MahtaMirzaei** [comment]
Ops! I made a mistake pushing my new file to the same branch as before. Can I fix that anyway?

I've got more questions from you, mostly about this new PR:
Would you rather I used the same method for applying stylesheets to each of the "Hello World"s? I mean something like using an id for each one of them or using a paragraph tag for them all, not headers.

Also, I did realize that by background-color you meant it for the whole page, not each text, but it's not like any page could have 6 different background colors, so I featured the texts with backgrounds as well. Is that OK?

**Comment by mkermani144** [comment]
> the HTML files weren't in my folder until you allowed the HTML branch to merge to main.

How is that possible? If you created `CSS` branch from `HTML`, the files of the old branch for sure exist in the new branch and shouldn't be included in the diff of this PR. You need to investigate what happened exactly (e.g. try a `git diff` on your local repo and see if the HTML file is part of the diff).

If the issue persists, we can investigate it together.

**Comment by mkermani144** [comment]
> Ops! I made a mistake pushing my new file to the same branch as before. Can I fix that anyway?
> 
> I've got more questions from you, mostly about this new PR: Would you rather I used the same method for applying stylesheets to each of the "Hello World"s? I mean something like using an id for each one of them or using a paragraph tag for them all, not headers.
> 
> Also, I did realize that by background-color you meant it for the whole page, not each text, but it's not like any page could have 6 different background colors, so I featured the texts with backgrounds as well. Is that OK?

Yes, but it's a bit complex. Anyways, I prefer you learn about these stuff and fix it, leaving this PR for #2 only.

The steps to fix the issue:
1. create a new branch from this branch, e.g. `css-colors`
2. reset this branch to a previous commit hash, where new changes related to #3 are not present (`git reset --hard <hash>`)
3. do a force push (`git push -f`)

If you have any questions, etc. feel free to ask. Also please note that force pushing is rarely recommended and here is one of the few cases where its acceptable.

**Comment by mkermani144** [comment]
I prefer to review your PR after the above issues are fixed.

**Comment by MahtaMirzaei** [comment]
> > the HTML files weren't in my folder until you allowed the HTML branch to merge to main.
> 
> How is that possible? If you created `CSS` branch from `HTML`, the files of the old branch for sure exist in the new branch and shouldn't be included in the diff of this PR. 
> 
Actually, I was really trying not to mix things up, thinking we must create new branches from the main branch. So, after I completed what I wanted to push for #2 , I moved the HTML folder out of the repo's folder, then started creating a CSS branch and everything. I believe this may have caused the problem.

**Comment by MahtaMirzaei** [comment]
> > the HTML files weren't in my folder until you allowed the HTML branch to merge to main.
> 
>You need to investigate what happened exactly (e.g. try a `git diff` on your local repo and see if the HTML file is part of the diff).
> 
I tried git diff on my local repo and it showed me nothing at all.

**Comment by MahtaMirzaei** [comment]
> > Ops! I made a mistake pushing my new file to the same branch as before. Can I fix that anyway?
> > I've got more questions from you, mostly about this new PR: Would you rather I used the same method for applying stylesheets to each of the "Hello World"s? I mean something like using an id for each one of them or using a paragraph tag for them all, not headers.
> > Also, I did realize that by background-color you meant it for the whole page, not each text, but it's not like any page could have 6 different background colors, so I featured the texts with backgrounds as well. Is that OK?
> 
> Yes, but it's a bit complex. Anyways, I prefer you learn about these stuff and fix it, leaving this PR for #2 only.
> 
> The steps to fix the issue:
> 
>     1. create a new branch from this branch, e.g. `css-colors`
> 
>     2. reset this branch to a previous commit hash, where new changes related to [CSS: colors #3](https://github.com/ConnecMent/inception/issues/3) are not present (`git reset --hard <hash>`)
> 
>     3. do a force push (`git push -f`)
> 
> 
> If you have any questions, etc. feel free to ask. Also please note that force pushing is rarely recommended and here is one of the few cases where its acceptable.

All done. I just used  git reset --hard HEAD@{n}  instead.

**Comment by MahtaMirzaei** [comment]
Now I've git another tiny problem. I can't commit anything on branch css-colors and it's not even showed on the site.

**Comment by MahtaMirzaei** [comment]
> Now I've got another tiny problem. I can't commit anything on branch css-colors and it's not even showed on the site.

I pushed once and this was solved

**Comment by mkermani144** [comment]
The changes still contains code unrelated to the linked issue. Please do EXACTLY what is asked in the issue. It's not important in issues focused on learning at all, but you need to learn real world methods of doing tasks.

- do not include unwanted html and css
- do not give tags other than `<html>` a class
- etc.

**Comment by MahtaMirzaei** [comment]
> The changes still contains code unrelated to the linked issue. Please do EXACTLY what is asked in the issue. It's not important in issues focused on learning at all, but you need to learn real world methods of doing tasks.
> 
>     * do not include unwanted html and css
> 
>     * do not give tags other than `<html>` a class
> 
>     * etc.

Done.

File: mahta/CSS/CSS basics/index2.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+<head>
```
Comment (mkermani144) [inline mahta/CSS/CSS basics/index2.html:3]: As I mentioned earlier, based on the issue, you need to add class to this tag, not the body tag.

File: mahta/CSS/CSS basics/index2.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+<link rel="stylesheet" href="style.css">
+</head>
+<body class="bor">
+<p> This is a page with red background:</p>
```
Comment (mkermani144) [inline mahta/CSS/CSS basics/index2.html:7]: These are unwanted HTML which are not mentioned in the issue.

File: mahta/CSS/CSS basics/style.css [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
```
Comment (mkermani144) [inline mahta/CSS/CSS basics/style.css:1]: 1. Removing redundant empty lines is highly preferred.
2. As I mentioned earlier, please do not include whitespace characters in file and directory names.

**Review by mkermani144 — CHANGES_REQUESTED** [review]

File: mahta/CSS/CSS basics/style.css (hunk 3 above)
Comment (MahtaMirzaei) [reply mahta/CSS/CSS basics/style.css]: I fixed everything I thought I should. I'm sorry for the problems you had to mention more than once and I'll try to pay better attention.

**Review by mkermani144 — APPROVED** [review]

**Pull request merged**